Analysis/HiggsTauTau/src/HTTCategories.cc

```diff
@@ -283,15 +283,23 @@ namespace ic {
       outtree_->Branch("m_vis",             &m_vis_.var_double);
       outtree_->Branch("pt_h",              &pt_h_.var_double);
       outtree_->Branch("pt_tt",             &pt_tt_.var_double);
```

Comment:
You're not filling this variable/branch anymore, right? Can probably get rid of both the branch in the tree and the variable itself!